### ISSUE TITLE
[WoW] Item level, URL formatting, code cleanup

### ISSUE THREAD

**Issue by gdude2002** [UltrosBot/Ultros-contrib#21]
Two things from @turt2live on IRC..
- When the realm has a space in it, there's also a space in the URL. It should be URL-formatted.
- Item level should be shown

Additionally, the plugin code states that a `-` should be used instead of a space, but this is unnecessary as command arguments wrapped in `""` include the spaces within.

---

Manual assignee: @Zarthus - Feel free to work on this when you feel like you'll have time
##

**Comment by gdude2002** [comment]
Any luck, @Zarthus?

**Comment by Zarthus** [comment]
nope

**Comment by gdude2002** [comment]
It's been ~6 months, are you still busy?

**Comment by Zarthus** [comment]
This just doesn't feature very high among my priorities.

**Comment by gdude2002** [comment]
Sure, I get that, but it should be a ten-minute job?

**Comment by Zarthus** [comment]
No. I have to update the plugin for both another bot and this bot. Last time I checked it was broken so I would need to find out what changed in the API, fix the changes, implement the new features, test the changes, commit the changes, and write a pull request.

If you feel you can do it in ten minutes, feel free to do so.

**Comment by gdude2002** [comment]
Ah, okay, I wasn't aware the API had broken.
